DataFormats/HcalDetId/interface/HcalDetId.h

```diff
@@ -4,7 +4,12 @@
 #include <iosfwd>
 #include "DataFormats/DetId/interface/DetId.h"
 #include "DataFormats/HcalDetId/interface/HcalSubdetector.h"
+#include "FWCore/Utilities/interface/Exception.h"
 
+//
+// note: not able to use dynamic library linkage with cuda
+// https://docs.nvidia.com/cuda/cuda-compiler-driver-nvcc/index.html#libraries
```

Comment:
while the comment is true, I don't see why it is relevant here ?

Reply:
will remove...